samples/Samples.Authorization/Samples.Authorization.csproj

```diff
@@ -1,9 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
+  <Import Project="../../Samples.props" />
```

Comment:
I designed the sample projects to have a csproj that is valid by itself, based on the .NET template, as part of the sample.  The only required change is the reference to the nuget package rather than the project.  For that reason, I do not want to change this file any more than absolutely necessary.

The projects are designed to be as close to user-code as possible; users should be able to open a new template, and copy the code directly in and/or see the difference between the sample and the template.  I would like to update these samples with each new version of Visual Studio so that the sample matches the current template, and simply contains the few changes necessary for GraphQL.NET Server to operate.  That is the goal of the samples, and I feel this is important and valuable.

This is why there are no global using statements present.  It was done this way on purpose, specifically so that users can copy files from the sample directly into their project with no changes.  (Assuming their project is based on the current .NET template, of course.)

If we require some additional settings for the sample projects to facilitate the tests, let's add another `Directory.Build.props` in the samples folder, so that no additional tags are necessary within this csproj.

Reply:
Understandable but there is always some 'context' information to configure csprojs - `Directory.Build.props` for example how you mentioned before. Project configuration should be convenient for us as well. So I renamed `Samples.props` into nested `Directory.Build.props`.